spec/controllers/tasks_controller_spec.rb

```diff
@@ -252,6 +278,10 @@
         expect { put_update }.to change { task.reload.license.name }.to('new_license')
       end
 
+      it 'updates the tasks labels' do
+        expect { put_update }.to change { task.reload.labels }.to([new_label]).from([existing_label])
```

Comment:
I would put it the outer way around - sounds more natural:

```suggestion
        expect { put_update }.to change { task.reload.labels }.from([existing_label]).to([new_label])
```